pyproject.toml

```
@@ -56,6 +56,7 @@ perceval-mozilla = {branch = "master", git = "https://github.com/chaoss/grimoire
 perceval-opnfv = {branch = "master", git = "https://github.com/chaoss/grimoirelab-perceval-opnfv"}
 perceval-puppet = {branch = "master", git = "https://github.com/chaoss/grimoirelab-perceval-puppet"}
 perceval-finos = {branch = "master", git = "https://github.com/Bitergia/grimoirelab-perceval-finos"}
+perceval-weblate = {branch = "master", git = "https://github.com/Bitergia/grimoirelab-perceval-weblate"}
```

I think the repository is moved to the CHAOSS organization. I think we can update the URL.

```suggestion
perceval-weblate = {branch = "master", git = "https://github.com/chaoss/grimoirelab-perceval-weblate"}
```